components/form-control/demo/demo.js

```diff
@@ -5,7 +5,6 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import FormControl from '../umd/form-control.development.js';
```

Comment:
Maybe `import { FormControl } from '../script.js'` ?

Reply:
and remove the script import for the dist bundle of the form control? 
```
    <script src="../dist/form-control.development.js"></script>
```

Reply:
It won't work because webpack will bundle the components library, but it won't be as a umd module which is what the dependencies loaded in demo.html expect. They will look for external dependency "components" but it will be undefined as it was bundled in the FormControl. I didn't want to change the form control to import the dependencies in script.js so I left it like this.